tests/cases/fourslash/navigationBarJsDoc.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="fourslash.ts"/>
+
+// @Filename: foo.js
+/////** @typedef {(number|string)} NumberLike */
+/////** @typedef {(string|number)} */
+////const x = 0;
+
+verify.navigationBar([
+  {
+    "text": "NumberLike",

```

Comment:
The current implementation is different for javascript, so that will be something fixed by #8958.